solana/pyth2wormhole/client/src/util.rs

```diff
@@ -84,7 +84,7 @@ impl<T> RLMutex<T> {
     pub fn new(val: T, rl_interval: Duration) -> Self {
         Self {
             mtx: Mutex::new(RLMutexState {
-                last_released: Instant::now() - rl_interval,
+                last_released: Instant::now().checked_sub(rl_interval).unwrap(),
```

Comment:
I guess? Underflows should be impossible for any reasonable value (i.e. lower than current unix timestamp value, which is decades worth)

Reply:
yeah I checked at that is nearly impossible to fail. So I think unwrap is fine.